src/pip/_internal/operations/build/wheel.py

```diff
@@ -26,8 +26,8 @@ def build_wheel_pep517(
     assert metadata_directory is not None
     if build_options:
         # PEP 517 does not support --build-options
-        logger.error('Cannot build wheel for %s using PEP 517 when '
-                     '--build-option is present' % (name,))
+        logger.error('Cannot build wheel for {} using PEP 517 when '
+                     '--build-option is present'.format(name))
```

Comment:
I must be missing something. Isn’t this the kind of the wrong call `flake8-logging-format` is supposed to reject?

Reply:
The plugin has a rule for it. It's not been opted into, IIUC.